src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/PersonSex.java

```diff
@@ -0,0 +1,53 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+/**
+ * Example PDI+2'
+ */
+@Getter
+@EqualsAndHashCode(callSuper = false)
+@Builder
+public class PersonSex extends Segment {
+    protected static final String KEY = "PDI";
+
+    @NonNull
+    private final Gender gender;
+
+    @Override
+    public String getKey() {
+        return KEY;
+    }
+
+    @Override
+    public String getValue() {
+        return gender.getCode();
+    }
+
+    @Override
+    protected void validateStateful() throws EdifactValidationException {
+    }
+
+    @Override
+    public void preValidate() throws EdifactValidationException {
```

Comment:
Should we add a `null` or `isBlank` check here to make sure that there's a value in this segment as it's used by `toEdifact`?

Reply:
I'm on the fence about this as `gender` has a `@NonNull` annotation against it, so i don't see a way of constructing this segment without the gender being set

Reply:
Also with gender being an enum, the value has to be equal to one of the predefined values

Reply:
Yeah I agree especially with the `@NonNull` annotation